src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -336,6 +336,7 @@ const useGroupInfo = () => {
       setIsLoadingInitChat(true);
       await handleAnonymousMessage(selectedUser);
     } catch (e) {
+      console.info('error anon', e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While it's good to have error handling, logging errors to the console is not always the best approach. It would be better to handle the error in a way that provides feedback to the user or logs the error to an external service where it can be monitored and addressed.
```diff
- console.info('error anon', e);
+ // replace with your preferred error handling method
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->